packages/mdl-radio/README.md

```diff
@@ -0,0 +1,183 @@
+```
+
+> NOTE: We plan on creating a generic `mdl-form-field` wrapper class - as seen within the demo -
+> that can be used to position form controls and labels, as well as make them RTL-aware.
+> See https://github.com/google/material-design-lite/issues/4775.
+
+#### Disabled Radios
+
+```html
+<div class="mdl-radio mdl-radio--disabled">
+  <input class="mdl-radio__native-control" type="radio" id="radio-1"
+         name="radios" aria-labelledby="radio-1-label" disabled>
+  <div class="mdl-radio__background">
+    <div class="mdl-radio__outer-circle"></div>
+    <div class="mdl-radio__inner-circle"></div>
+  </div>
+</div>
+<label id="radio-1-label" for="radio-1">Disabled Radio 1</label>
+```
+
+Note that `mdl-radio--disabled` is necessary on the root element in order to avoid having the ripple
+elements intercept pointer events when using JS.
+
+### Using the JS Component
+
+MDL Radio ships with Component / Foundation classes which provide enhanced interaction UX via
+[mdl-ripple](../mdl-ripple), as well as APIs for programmatically altering the radio's state.
+
+#### Including in code
+
+##### ES2015
+
+```javascript
+import MDLRadio, {MDLRadioFoundation} from 'mdl-radio';
+```
+
+##### CommonJS
+
+```javascript
+const mdlRadio = require('mdl-radio');
+const MDLRadio = mdlRadio.default;
+const MDLRadioFoundation = mdlRadio.MDLRadioFoundation;
+```
+
+##### AMD
+
+```javascript
+require(['path/to/mdl-radio'], mdlRadio => {
+  const MDLRadio = mdlRadio.default;
+  const MDLRadioFoundation = mdlRadio.MDLRadioFoundation;
+});
+```
+
+##### Global
+
+```javascript
+const MDLRadio = mdl.Radio.default;
+const MDLRadioFoundation = mdl.Radio.MDLRadioFoundation;
+```
+
+#### Fully-automatic: DOM Rendering + Initialization
+
+```javascript
+const root = MDLRadio.buildDom({id: 'my-radio', labelId: 'my-radio-label'});
+const radio = MDLRadio.attachTo(root);
+// append root to element, etc...
+```
+
+You can use `MDLRadio.buildDom` to dynamically construct radio DOM for you.
+`MDLRadio.buildDom` takes an options object with values described below:
+
+| Option | Type | Default | Description |
+| --- | --- | --- | --- |
+| `id` | `string` | `mdl-radio-<unique_id>` | The id for the native radio control. |
+| `labelId` | `string` | `mdl-radio-label-<id_value>` | The id of the element which label's this radio. The default will use the `id` param and prefix it with `mdl-radio-label`. This value is attached to the `aria-labelledby` attribute on the native control. |
+| `groupId` | `string` | '' | When supplied, this will be given to the "name" attribute on the native radio control, allowing you to specify that a collection of buttons are part of the same radio group. |
+
+> **NOTE**: Regardless of how you instantiate a radio element, you should always strive to
+> provide an id for the radio that's used within its label's `for` attribute, as well as an id
+> for its label which is used in the native control's `aria-labelledby` attribute. This will ensure
```

Comment:
Ah that's a great point! I think I was doing this for checkboxes, which I can submit a fix for.